openassessment/xblock/lms_mixin.py

```diff
@@ -6,14 +6,14 @@
 
 class GroupAccessDict(Dict):
     """Special Dict class for serializing the group_access field"""
-    def from_json(self, access_dict):  # pylint: disable=arguments-differ
```

Comment:
Pylint complaining that the args of the overridden function differed from the original function signature.

Reply:
Idea: add a line `access_dict = value` and then eave the rest of the function unchanged, for readability

Reply:
It would make the internals more readable, but I don't love creating and assigning a new variable just for aliasing.

Reply:
+1 for avoiding the alias.

It makes it easier to jump around inherited methods/classes, when the variables are homogeneous.